src/github.com/getlantern/flashlight/config/config.go

```diff
@@ -338,3 +341,12 @@ func (updated *Config) updateFrom(updateBytes []byte) error {
 	}
 	return nil
 }
+
+func inHomeDir(filename string) string {
+	log.Tracef("Determining user's home directory")
+	usr, err := user.Current()
+	if err != nil {
+		panic(fmt.Errorf("Unable to determine user's home directory: %s", err))
```

Comment:
panic and log.Fatalf are different.  log.Fatalf actually calls os.Exit, from which one cannot recover. Panics are in theory recoverable.  Right now, we don't recover on any panics, but we might someday, so except for stuff in the main routine, it's probably best to panic instead of using log.Fatalf.